tla/tlc.sh

```diff
@@ -3,7 +3,7 @@
 # Licensed under the Apache 2.0 License.
 # Original License below
 # Adapted from: https://github.com/pmer/tla-bin
-exec java -XX:+IgnoreUnrecognizedVMOptions -XX:+UseParallelGC -Xms2G -Xmx2G -cp tla2tools.jar:CommunityModules-deps.jar tlc2.TLC "$@" -workers auto
+exec java -XX:+IgnoreUnrecognizedVMOptions -XX:+UseParallelGC -Xms2G -Xmx2G -cp tla2tools.jar:CommunityModules-deps.jar tlc2.TLC "$@" -workers auto -fpmem 1
```

Comment:
It seems bizarre that we have to specify all these separate memory-related options on a relatively recent jvm...

Reply:
We don't, I think we can tidy this up

Reply:
Let's find out #5700